dts/bindings/adc/atmel,sam0-adc.yaml

```diff
@@ -43,6 +43,18 @@ properties:
       enum:
         - 0
         - 14
+    pinctrl-0:
+      type: phandles
+      description: |
+        PIO pin configuration for adc pin and channel.  We expect that
+        the phandles will reference pinctrl nodes.  These nodes will
+        have a nodelabel that matches the Atmel SoC HAL defines and
+        be of the form p<port><pin><periph>_<chan>_<signal>.
+
+        For example the I2C0 on SAMD21 would be
+           pinctrl-0 = <&pa9d_adc_chan19>;
```

Comment:
Is an "_" missing between chan and 19 or is the syntax stated in line 52 wrong?

Reply:
It still is confusing but p a=port 9=pin d=periph inst=adc and chan=chan19

Reply:
would have made it `_chan<channel_number>` but well its fine since there is an example